### ISSUE TITLE
Prise en compte biogaz

### ISSUE THREAD

**Issue by SonephetR2** [datagir/nosgestesclimat#982]
J'ai « audité » l'impact CO2 d'un basculement sur du biogaz quand je suis passé chez Ekwateur : émissions passant à 13 g CO₂ éq. par kWh, soit -94.4% par rapport à Engie. L'impact est conséquent, ce serait bien de prévoir ce genre d'alternatives dans le calcul.

**Comment by laem** [comment]
C'est en effet un point à améliorer, mais nous n'avons pas encore fouillé suffisamment dans la documentation disponible. Voir #4. C'était la 4ème issue, il y en a maintenant 1000 :sweat_smile:

**Comment by SonephetR2** [comment]
Effectivement ! C'est un sacré effort à fournir ! N'hésitez pas à spécifier quel serait le type de documentation à fournir. Je suis prêt à fouiller un peu plus parce que c'est un sujet qui semble mériter une analyse approfondie et qui m'intéresse, d'autant qu'il semble lié à beaucoup d'autres enjeux environnementaux non climatiques, et pas forcément du « bon côté ».

**Comment by laem** [comment]
La référence semble être l'étude de carbone 4 : http://www.carbone4.com/wp-content/uploads/2019/06/Publication-Carbone-4-biométhane-et-climat.pdf
Est-ce celle qui vous a donné le chiffre de -95% ?

**Comment by SonephetR2** [comment]
Le mix moyen de production de biométhane peut être retrouvé [ici](https://bilans-ges.ademe.fr/documentation/UPLOAD_DOC_FR/index.htm?gaz2.htm), avec comme source principale [ceci](https://www.grdf.fr/documents/10184/1502679/Evaluation+des+impacts+GES+de+l%E2%80%99injection+du+biom%C3%A9thane+dans+les+r%C3%A9seaux+rapport+final+et+r%C3%A9sum%C3%A9+07.04.2015.pdf/d1df4981-c7dc-460d-ad97-22a02d7eaa4a). Les cultures dédiées et CIVE (cultures intermédiaires à vocation énergétique – qui ont l'air agronomiquement intéressantes) représentent effectivement environ 7% et 7% respectivement du total en 2018. Le chiffre retenu par l'ADEME est expliqué dans le détail dans ce dernier document.

**Comment by laem** [comment]
Merci ! 

## Au sujet du facteur d'émission 

La base carbone retient maintenant `0.0444 kgCO2e/kWhPCI`. 

<details><summary>Leur justification est dans la documentation [déplier]</summary>

Pour des questions méthodologiques, cette approche n’est pas compatible avec la logique de construction des facteurs d’émissions de la Base Carbone® et ne peut être utilisée pour la réalisation d’un bilan d’émissions de GES. Ainsi, des travaux complémentaires ont été menés aboutissant à une mise à jour de l'Analyse de Cycle de Vie en 2020 [[144]](https://bilans-ges.ademe.fr/documentation/UPLOAD_DOC_FR/references.htm) et à un facteur d’émissions de 44,1 gCO2e / kWh PCI. Cette dernière valeur traduit l’impact GES induit par la production d’un kWh de biométhane, contrairement à la valeur de 23,4 gCO2e / kWh PCI issue de la précédente étude, qui traduit l’impact global de la filière de méthanisation et injection, intégrant impacts induits et réductions d’émissions de GES apportés par cette filière. </details>

Je dois dire que je ne comprends pas bien ce qui nous empêcherait d'utiliser le facteur le plus faible dans le cadre de ce modèle de calcul consommation. 

## Au sujet de l'utilisation des terres

Ton lien et celui de la base carbone vers l'étude de GRDF n'est plus en ligne. [La voici en PDF](https://github.com/datagir/nosgestesclimat/files/9542516/evaluation-des-impacts-ges-biomethane-_synthese-quantis_2017.pdf). On peut y voir : 

![image](https://user-images.githubusercontent.com/1177762/189527416-4e8a3725-ef5a-448e-8557-0ad0007e0720.png)

Dont la part rouge est détaillée ici : 

![image](https://user-images.githubusercontent.com/1177762/189527431-3fdcc5b2-22c0-49e1-b5f0-68762f780d57.png)

On peut y voir que les cultures dédiées sont très faibles en proportion, ce qui est plutôt très rassurant. 

Cela dit, et je ne sais pas trop comment le définir précisément, j'ai quand même une forte impression qu'une grande partie de ces composants pourraient être valorisés différemment : concrètement, on crame et on valorise donc à un prix vendu très élevé (le gaz en prix de gros [coûte en septembre 2022](https://twitter.com/i/timeline) près de 2 fois le prix de production du biogaz (90€/MWh)) les restes de poulet de la cantine qu'on aurait pu juste ne pas produire (bleu); la pelouse domestique que le voisin tond toutes les semaines (bleu clair); le fumier de cultures qui auraient pu être dédiées à du végétal humain (mauve); etc. L'étude pourtant semble très complète, donc c'est à creuser. 

Une autre question dont la réponse y est peut-être est celle-ci : la filière peut-elle faire x10 en production sans introduire des externalités négatives pour l'instant maitrisées ? Ce serait intéressant d'y répondre pour l'utilisateur de NGC, lui montrer que son choix peut passer à l'échelle de la société. Carbone4 semble y répondre négativement : le remplacement du gaz fossile par le biométhane ne doit pas conduire à une non réduction de la consommation de gaz en général, et encore moins à un effet rebond ! 

![image](https://user-images.githubusercontent.com/1177762/189527887-931ee20a-8ad6-431f-bd4d-d705fa9eefca.png)

Un **avertissement** pourrait donc être intégré pendant la simulation pour évoquer cet impossible passage à l'échelle : 

> Le biométhane réduit **énormément** les émissions de CO2 du gaz que vous consommez, mais la production actuelle est **incapable de passer à l'échelle** : la réduction globale de toute consommation de gaz reste **essentielle**, que l'on consomme du biogaz ou du gaz fossile.

**Comment by Benjamin-Boisserie-ABC** [comment]
@laem au sujet du FE biogaz mix moyen de la Base Carbone et des raisons du message d'avertissement de l'ADEME. L'étude réalisée intègre "_à la fois les émissions de GES liées à la production du biométhane, mais également les émissions GES évitées par la méthanisation au sein des secteurs agricoles et des déchets_", ce qui est contraire au standard de comptabilité carbone pour la réalisation de BEGESr et de Bilan Carbone, qui impose de différencier les deux. Les émissions évitées sont intéressantes à comptabilisées mais de manière séparées. Elles ne peuvent en aucun cas être soustraites des émissions induites afin de réduire le volume global d'émissions. C'est pourquoi l'ADEME met ce message d'avertissement. Il convient donc de prendre le FE mix moyen de 44,1 gCO2e / kWh PCI
Pour le reste je vais regarder mais en effet, le biogaz ne semble pas le plus adapté pour chauffer son logement. A voir comment on porte cette info à l'utilisateur

**Comment by laem** [comment]
> Elles ne peuvent en aucun cas être soustraites des émissions induites afin de réduire le volume global d'émissions.

Oui, j'ai bien compris que l'ADEME s'empechait de l'utiliser dans la base carbone, mais ma question concerne le modèle NGC, qu'est-ce qui nous empêche de l'utiliser ici précisément ?

**Comment by Benjamin-Boisserie-ABC** [comment]
Car le modèle NGC suit la même philosophie qui cherche à évaluer les émissions réellement induites par nos actes et les structures qui les supportent et non les émissions évitées par certaines pratiques. 

Ainsi, ici la réalité c'est que la structure technique permettant une consommation de biogaz contribue à émettre 44,1 gCO2e / kWh PCI. Ça ce sont les émissions réellement induites. Les émissions évitées ajoutées qui permettent de faire descendre le facteur à 23,9 gCO2e / kWh PCI ne sont que des prévisions (et non une réalité même imparfaitement estimées) qui plus est extrêmement sensibles, comme toutes prévisions d'émissions évitées, aux hypothèses considérées.

Ainsi, quand il est écrit que l'étude décide déduire aux émissions induites "_les émissions GES évitées par la méthanisation au sein des secteurs agricoles et des déchets (par exemple, par la réduction des émissions liées au traitement des effluents d’élevages, ou par la réduction des émissions liée à l’utilisation du digestat à la place d’engrais industriels)_", ces émissions évitées ne sont ici qu'une potentialité et non une réalité car l'étude à chercher à "_quantifier la contribution de cette filière aux objectifs de réduction des émissions de GES"_

Après je suis d'accord sur le message de contextualisation à apporter à l'utilisateur. Il ne suffit pas de de remplacer du gaz fossile par du biogaz pour toutes les raisons que tu as identifié (gisement insuffisant, privilégier le biogaz pour les usages n'ayant pas d'alternatives, passage à l'échelle avec externalités négatives potentielles avec en premier lieu un usage du sol à finalité énergétique).

**Comment by Clemog** [comment]
De mon côté, je suis favorable à l'option biogaz, sur le même fonctionnement que pour les biiocarburants : https://github.com/datagir/nosgestesclimat/pull/1396

J'ai l'impression que cette issue documente davantage les questions de production et d'implication de l'utilisation du biogaz c'est très intéressant, on pourrait écrire un petit article de blog sur le sujet

**Comment by laem** [comment]
> on pourrait écrire un petit article de blog sur le sujet

Autre option moins "low cost" : le mettre dans la doc, s'assurer que c'est lisible et citer la doc dans la release. 
Mais oui ce serait sympa de transformer /nouveautés en blog avec des articles thématiques.
